chef/cookbooks/hyperv/recipes/openstack_install_neutron.rb

```diff
@@ -2,7 +2,7 @@
 
 cookbook_file "#{node[:cache_location]}#{node[:openstack][:neutron][:file]}" do
   source node[:openstack][:neutron][:file]
-  not_if { ::File.exists?(node[:openstack][:neutron][:installed]) }
+  not_if { ::File.exist?(node[:openstack][:neutron][:file]) }
```

Comment:
but the point of the [:installed] attribute is to guard multiple runs. so the attribute needs fixing to a file that actually exists.